site/AntDesign.Docs/Demos/Components/DatePicker/demo/RangePickerDemo.razor

```diff
@@ -11,13 +11,28 @@
 <RangePicker TValue="DateTime?[]" Picker="@DatePickerType.Quarter" />
 <br />
 <RangePicker TValue="DateTime?[]" Picker="@DatePickerType.Year" />
-
+<br />
 <RangePicker TValue="DateTime?[]" ShowTime='@("HH:mm")' OnChange="OnTimeRangeChange" />
+<br />
+<br />
+Custom Disabled Only:
+<br />
```

Comment:
Please put the showcase to `DisabledDate.razor`.

Reply:
Removed demo since it now uses default always